tests/__init__.py

```diff
@@ -240,10 +240,14 @@ def test_update(self):
     def test_delete(self):
         plugin = self.model_class.objects.order_by("?").first().pk
         response = self.client.delete(f"{self.URL}/{plugin}")
-        self.assertEqual(response.status_code, 405, response.json())
+        if "playbook" in self.URL:
+            self.assertEqual(response.status_code, 204, response)
+        else:
+            self.assertEqual(response.status_code, 405, response.json())
```

Comment:
instead of having this `if` condition, just override the method in the `PlaybookConfigViewSet` test class and change the `assertEqual`